objects/src/block/header.rs

```diff
@@ -186,6 +186,11 @@ impl BlockHeader {
         self.timestamp
     }
 
+    /// Returns the block number of the epoch block to which this block belongs.
+    pub fn get_epoch_block_num(&self) -> u32 {
```

Comment:
I think I'd just name this `epoch_block_num()` since it's a very basic computation over the block num.

Reply:
I think the first suggestion matches better with the rest of the methods in `BlockHeader`